generator/resources/ec2_windows_performance_test_matrix.json

```diff
@@ -19,5 +19,26 @@
         "arc": "amd64",
         "valuesPerMinute": 5000,
         "family": "windows"
+      },
+      {
+        "os": "win-2025",
+        "ami": "cloudwatch-agent-integration-test-win-2025*",
+        "arc": "amd64",
+        "valuesPerMinute": 100,
+        "family": "windows"
+      },
+      {
+        "os": "win-2022",
```

Comment:
should this be win-2025? same on line 38

Reply:
fixed!